app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,51 @@
+<%  if current_user.present? %>
+  <nav class="navbar navbar-expand-sm navbar-dark bg-dark">
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExample03" aria-controls="navbarsExample03" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <div class="collapse navbar-collapse" id="navbarsExample03">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item">
+          <%= link_to 'Home', root_path, class: 'nav-link' %>
+        </li>
+        <li class="nav-item">
+          <%= link_to 'Notes', user_notes_path(current_user.id), class: 'nav-link' %>
+        </li>
+        <li class="nav-item">
+          <%= link_to 'Tasks', user_tasks_path(current_user.id), class: 'nav-link' %>
+        </li>
+      </ul>
+      <form class="form-inline my-2 my-md-0">
+        <input class="form-control" type="text" placeholder="Search">
+      </form>
+      <ul class="navbar-nav mr-5 pr-3">
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle text-white" href="http://example.com" id="dropdown03" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Settings</a>
+          <div class="dropdown-menu mr-5" aria-labelledby="dropdown03">
+            <%= link_to 'Log Out', session_path(:user_id), class: "dropdown-item",data: {turbo_method: :delete, turbo_confirm: "Are you sure?"} %>
```

Comment:
space missing

Reply:
Ammar bhae says don't to use spaces for HTML tags

Reply:
done